pkgs/native_assets_cli/lib/src/code_assets/code_asset.dart

```diff
@@ -87,7 +87,7 @@ final class CodeAsset {
     required LinkMode linkMode,
     required OS os,
     Uri? file,
```

Comment:
This is a style question, and not something for the current PR, but I personally like setting nullable parameter as `required` to make people think about if they want to leave out the argument.

Reply:
I do like that idea in this context. It means that the semantic layer around it explicitly has to pass null. And most likely if we're adding a new optional field, we should not be forgetting to pass it from the semantic API to the syntax.